sections/tooling/babel-plugin.md

```diff
@@ -124,12 +124,50 @@ It utilizes a babel helper to tag each styled component and library helper with
 
 ### Template String Transpilation
 
-We transpile `styled-components` tagged template literals down to a smaller representation than what Babel normally does, because `styled-components` template literals don't need to be 100% spec compliant.
+This plugin transpiles `styled-components` tagged template literals down to a smaller representation than what Babel normally does.
 
-Read more about [Tagged Template Literals](/docs/advanced#tagged-template-literals) in
-our dedicated section explaining them.
+Wait, transpiling tagged template literals? Doesn't Babel do this natively? 🤔
+
+You're currently using Babel to transpile your ES2015 JavaScript to ES5-compliant code. One of your presets (`es2015`/`env`/`latest`) includes the `babel-plugin-transform-es2015-template-literals` transform to make tagged template literals work in older browsers, but there is a caveat. Output of that plugin is quite wordy. It's done this way to meet specification requirements.
+
+Here's an example of the transpiled code processed with `babel-preset-latest`:
+
+```js
+var _templateObject = _taggedTemplateLiteral(['width: 100%;'], ['width: 100%;'])
+function _taggedTemplateLiteral(strings, raw) {
+  return Object.freeze(Object.defineProperties(strings, { raw: { value: Object.freeze(raw) } }))
+}
+var Simple = _styledComponents2.default.div(_templateObject)
+```
+
+`styled-components` styling code does not require full spec compatibility. This plugin will transpile template literals attached to styled-component to a slightly different form which still works in older browsers but has a much smaller footprint.
+
+Now here's an example of the code including `babel-plugin-styled-components` with the `{ transpileTemplateLiterals: true }` option:
 
-You can use the `transpileTemplateLiterals` option to turn this feature off.
+```js
+var Simple = _styledComponents2.default.div(['width: 100%;'])
+```
+
+Take a note that it will keep other template literals not related to styled-components as is:
+
+```js
+// Following will be converted:
+styled.div``
+keyframe``
+css``
+
+// But this will not be converted:
+`some text`
```

Comment:
@probablyup I noticed you (or maybe IDE formatter/prettier) _recently_ changed some of this example block's backticks to single quotes.

They _need_ to be backticks as that's what the codeblock is talking about; the fact that template strings without a "styled-component factory tag" aren't affected by the transpilation. Just fyi 🙂

Reply:
Prettier, yeah.